lib/driver.js

```diff
@@ -229,6 +229,12 @@ class XCUITestDriver extends BaseDriver {
       this.logEvent('customCertInstalled');
     }
 
+    if (this.opts.enableAsyncExecuteFromHttps && !this.isRealDevice()) {
+      await this.opts.device.shutdown();
```

Comment:
maybe a comment here explaining why we need to shut down the sim would be important; i was initially confused until i guessed that the seem needs to be started with the cert installed?